app/javascript/chat/__tests__/compose.test.jsx

```diff
@@ -87,6 +90,16 @@ describe('<Compose />', () => {
       expect(submitWithMessage).toEqual(false);
       expect(textfieldIsEmpty).toEqual(true);
     });
+
+    it('should pressed enter and shift', () => {
```

Comment:
**nitpick (non-blocking):** I think this message could be worded a bit differently, but we can do that later.